### PR TITLE
Add supplemental notebooks to June site

### PR DESCRIPTION
**Pull request by jashapiro**
I added a bit about installation and quoting column names. Nothing too much...

I considered adding more about labeling points on plots, but I thought we might have done enough there in the Slack messages...

### CODE REVIEW

File: supplemental_notebooks/2021-06-28_muddiest_points.Rmd [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+If we want to select an individual column as a **vector**, we can use the `$`:
+
+```{r}
+bill_lengths <- penguins$bill_length_mm
+
+# look at the structure of the vector we extracted (so we don't print it all)
+str(bill_lengths)
+```
+
+Alternative, we could use a `tidyverse` function from the `dplyr` package called `pull()` to accomplish the same task.
```
Comment (cbethell) [inline supplemental_notebooks/2021-06-28_muddiest_points.Rmd:127]:
```suggestion
Alternatively, we could use a `tidyverse` function from the `dplyr` package called `pull()` to accomplish the same task.
```

File: supplemental_notebooks/2021-06-28_muddiest_points.Rmd [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+```{r}
+penguins %>% 
+  dplyr::select(species, island, year) %>%
+  head()
+```
+### Filtering
+
+We didn't cover it in the lecture, but it is possible to do filtering by rows in base R too.
+First let's look at a tidyverse example:
```
Comment (cbethell) [inline supplemental_notebooks/2021-06-28_muddiest_points.Rmd:157]: Can we maybe add sentence or comment here telling folks what we're filtering by so they know what to look out for in the results of the below chunk?

**Review by cbethell — APPROVED** [review]
LGTM! I really just have one comment below, but otherwise I do not believe there's a need for me to re-review

File: supplemental_notebooks/2021-06-28_muddiest_points.Rmd (hunk 2 above)
Comment (jashapiro) [reply supplemental_notebooks/2021-06-28_muddiest_points.Rmd]: Done!

**Pull request merged**